source/KeyMapping.cpp

```diff
@@ -7,13 +7,14 @@ KeyMapping::KeyMapping() {
 	map.set("strafeLeft", Array<GKey>{ (GKey)'a', GKey::LEFT });
 	map.set("moveBackward", Array<GKey>{ (GKey)'s', GKey::DOWN });
 	map.set("strafeRight", Array<GKey>{ (GKey)'d', GKey::RIGHT });
+	map.set("sprint", Array<GKey>{ GKey::LSHIFT });
 	map.set("openMenu", Array<GKey>{ GKey::ESCAPE });
 	map.set("quit", Array<GKey>{ GKey::KP_MINUS, GKey::PAUSE });
 	map.set("crouch", Array<GKey>{ GKey::LCTRL });
 	map.set("jump", Array<GKey>{ GKey::SPACE });
 	map.set("shoot", Array<GKey>{ GKey::LEFT_MOUSE });
 	map.set("scope", Array<GKey>{ GKey::RIGHT_MOUSE});
-	map.set("dummyShoot", Array<GKey>{ GKey::LSHIFT });
```

Comment:
We generally try to keep historical FPSci behavior the same unless  a new feature demands a change. Since enabling sprint is optional I would suggest not overriding the default `dummyShoot` as many experiments rely on this. Perhaps sprint could be `TAB` to start with.

Reply:
The argument against changing defaults is this will break compatibility of old experiments. I agree `LSHIFT` is common for sprint, but none of our old experiments rely on this while many rely on `LSHIFT` for `dummyShoot`. I suppose another way to do this would be to default sprint-mode to off and have both `dummyShoot` and `sprint` map to `LSHIFT`.

@joohwankimNV for input.

Reply:
Yes, my proposal is to overload `LSHIFT` and have sprint disabled by default. So it's mapped to `LSHIFT` it just doesn't do anything in the default setting until enabled.

It shouldn't break any compatibility by doing that since we aren't changing the old defaults, just mapping another key to it that does nothing by default.

Reply:
I've made the necessary changes and pushed them. Let me know if they look okay or not. I have changed 'dummyShoot' back to LSHIFT and kept sprinting to LSHIFT as well.